typings/index.d.ts

```diff
@@ -0,0 +1,93 @@
+declare module 'reduxful' {
+  // fetchUtils
+  export interface RequestAdapterOptions {
+    method: string;
+    url: string;
+    headers: Map<string, string>;
```

Comment:
Echoing @andyfleming on this; it should be a `{ [header: string]: string }`.

Reply:
Yeap thanks! - I will take a look at the props once again, but I may need to replace every `Map<K,T>` references with indexed obj type 👍